Source/GUI/Qt/App.cpp

```diff
@@ -43,6 +43,11 @@ int main (int argc, char** argv)
     Core C;
     C.Errors_Continue=true; //Always for the GUI
 
+    // QApplication initialisation reset charset to default (ASCII when launched from finder) value.
+    #ifdef __MACOSX__
+    setlocale(LC_CTYPE, "UTF-8");

```

Comment:
We have `setlocale(LC_ALL, "");` just before, doesn't it work by doing the same command again? I am reluctant to force a specific charset.

Reply:
setlocale(LC_ALL, "") Applies the locale from the environment but when launched from the finder there isn't any.

So the default ANSI-C locale is used.

Reply:
Would it make sense to test `getlocale(LC_ALL, NULL)` (reading current locale) before forcing UTF-8, in case Apple puts another locale one day?